src/links.ts

```diff
@@ -25,6 +25,7 @@ const linksAttr = {
     'video',
   ],
   srcset: ['img', 'source'],
+  style: [''],
```

Comment:
Some initial thoughts - I kind of wonder out loud how valuable parsing inline CSS really is, if it doesn't also go down the rabbit whole and parse all css files linked via `<link>`.  The problem is - that leads me down a path of thinking why not other file formats?  

I could see this spiraling and getting out of scope pretty quickly.  I'm starting to think that a separate tool that does CSS scanning specifically may be a better bet here.

Reply:
It's a very good point. And I totally agree about CSS' rabbit hole. 

But inline styles is a good place to stop. 

Usually only `style="background-..."` is used like img for *presentation* images. I understand that linkinator is for checking generated documentation not the sites. But for example React SSR tools can generate style-embed images too in html markup.

Reply:
And CSS-in-tag-style is still HTML 'part' of crawling.